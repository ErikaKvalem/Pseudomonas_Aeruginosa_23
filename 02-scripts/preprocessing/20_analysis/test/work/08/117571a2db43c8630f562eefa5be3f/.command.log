
Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ forcats   1.0.0     ✔ readr     2.1.4
✔ lubridate 1.9.3     ✔ stringr   1.5.0
✔ purrr     1.0.2     ✔ tibble    3.2.1
Loading required package: grid
========================================
ComplexHeatmap version 2.14.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

New names:
• `` -> `...1`
Rows: 62 Columns: 10
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (1): gene_name
dbl (9): ...1, hAO_r1_ctrl, hAO_r1_inf, hAO_r2_ctrl, hAO_r2_inf, hAO_r3_ctrl...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 1358 Columns: 12
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr (3): gene_id, gene_name, genes_description
dbl (9): baseMean, log2FoldChange, lfcSE, stat, pvalue, padj, weight, log2Fo...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
 [1] "For this pathway there is only 1 DEG with padj<0.1. gene_name: ADAM17"   
 [2] "For this pathway there is only 1 DEG with padj<0.1. gene_name: AKAP12"   
 [3] "For this pathway there is only 1 DEG with padj<0.1. gene_name: AKAP8"    
 [4] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CCL2"     
 [5] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CD274"    
 [6] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CSF1"     
 [7] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CSF3"     
 [8] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CX3CL1"   
 [9] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL1"    
[10] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL3"    
[11] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL5"    
[12] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL6"    
[13] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL8"    
[14] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CYP1A1"   
[15] "For this pathway there is only 1 DEG with padj<0.1. gene_name: GCH1"     
[16] "For this pathway there is only 1 DEG with padj<0.1. gene_name: GJB2"     
[17] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IDO1"     
[18] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IFNAR1"   
[19] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IL36G"    
[20] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IRAK3"    
[21] "For this pathway there is only 1 DEG with padj<0.1. gene_name: JAK2"     
[22] "For this pathway there is only 1 DEG with padj<0.1. gene_name: KMO"      
[23] "For this pathway there is only 1 DEG with padj<0.1. gene_name: LITAF"    
[24] "For this pathway there is only 1 DEG with padj<0.1. gene_name: LYN"      
[25] "For this pathway there is only 1 DEG with padj<0.1. gene_name: MAP2K3"   
[26] "For this pathway there is only 1 DEG with padj<0.1. gene_name: MAPKAPK2" 
[27] "For this pathway there is only 1 DEG with padj<0.1. gene_name: NFKB1"    
[28] "For this pathway there is only 1 DEG with padj<0.1. gene_name: NFKBIB"   
[29] "For this pathway there is only 1 DEG with padj<0.1. gene_name: NOS2"     
[30] "For this pathway there is only 1 DEG with padj<0.1. gene_name: PTAFR"    
[31] "For this pathway there is only 1 DEG with padj<0.1. gene_name: PTGS2"    
[32] "For this pathway there is only 1 DEG with padj<0.1. gene_name: RELA"     
[33] "For this pathway there is only 1 DEG with padj<0.1. gene_name: RIPK2"    
[34] "For this pathway there is only 1 DEG with padj<0.1. gene_name: S100A8"   
[35] "For this pathway there is only 1 DEG with padj<0.1. gene_name: S100A9"   
[36] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SBNO2"    
[37] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SELENOS"  
[38] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SLC7A5"   
[39] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SMAD6"    
[40] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SOD2"     
[41] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SRC"      
[42] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TGFB1"    
[43] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TLR2"     
[44] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNFAIP3"  
[45] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNFRSF11A"
[46] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNFRSF1B" 
[47] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNIP1"    
[48] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNIP2"    
[49] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNIP3"    
NULL
