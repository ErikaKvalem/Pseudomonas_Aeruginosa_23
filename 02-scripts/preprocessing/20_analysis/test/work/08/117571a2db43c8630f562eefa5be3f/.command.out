 [1] "For this pathway there is only 1 DEG with padj<0.1. gene_name: ADAM17"   
 [2] "For this pathway there is only 1 DEG with padj<0.1. gene_name: AKAP12"   
 [3] "For this pathway there is only 1 DEG with padj<0.1. gene_name: AKAP8"    
 [4] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CCL2"     
 [5] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CD274"    
 [6] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CSF1"     
 [7] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CSF3"     
 [8] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CX3CL1"   
 [9] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL1"    
[10] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL3"    
[11] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL5"    
[12] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL6"    
[13] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CXCL8"    
[14] "For this pathway there is only 1 DEG with padj<0.1. gene_name: CYP1A1"   
[15] "For this pathway there is only 1 DEG with padj<0.1. gene_name: GCH1"     
[16] "For this pathway there is only 1 DEG with padj<0.1. gene_name: GJB2"     
[17] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IDO1"     
[18] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IFNAR1"   
[19] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IL36G"    
[20] "For this pathway there is only 1 DEG with padj<0.1. gene_name: IRAK3"    
[21] "For this pathway there is only 1 DEG with padj<0.1. gene_name: JAK2"     
[22] "For this pathway there is only 1 DEG with padj<0.1. gene_name: KMO"      
[23] "For this pathway there is only 1 DEG with padj<0.1. gene_name: LITAF"    
[24] "For this pathway there is only 1 DEG with padj<0.1. gene_name: LYN"      
[25] "For this pathway there is only 1 DEG with padj<0.1. gene_name: MAP2K3"   
[26] "For this pathway there is only 1 DEG with padj<0.1. gene_name: MAPKAPK2" 
[27] "For this pathway there is only 1 DEG with padj<0.1. gene_name: NFKB1"    
[28] "For this pathway there is only 1 DEG with padj<0.1. gene_name: NFKBIB"   
[29] "For this pathway there is only 1 DEG with padj<0.1. gene_name: NOS2"     
[30] "For this pathway there is only 1 DEG with padj<0.1. gene_name: PTAFR"    
[31] "For this pathway there is only 1 DEG with padj<0.1. gene_name: PTGS2"    
[32] "For this pathway there is only 1 DEG with padj<0.1. gene_name: RELA"     
[33] "For this pathway there is only 1 DEG with padj<0.1. gene_name: RIPK2"    
[34] "For this pathway there is only 1 DEG with padj<0.1. gene_name: S100A8"   
[35] "For this pathway there is only 1 DEG with padj<0.1. gene_name: S100A9"   
[36] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SBNO2"    
[37] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SELENOS"  
[38] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SLC7A5"   
[39] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SMAD6"    
[40] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SOD2"     
[41] "For this pathway there is only 1 DEG with padj<0.1. gene_name: SRC"      
[42] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TGFB1"    
[43] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TLR2"     
[44] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNFAIP3"  
[45] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNFRSF11A"
[46] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNFRSF1B" 
[47] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNIP1"    
[48] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNIP2"    
[49] "For this pathway there is only 1 DEG with padj<0.1. gene_name: TNIP3"    
NULL
